src/middleware/middleware.ts

```diff
@@ -45,5 +45,6 @@ const middleware: Middleware = async ({ app, query, route, $pinia }) => {
 
   const featureFlagStore = useFeatureFlagStore($pinia)
   featureFlagStore.initFromCookies(app.$cookies.get('features') ?? {})
+  featureFlagStore.initFromQuery(query)
```

Comment:
Can we make it only work on `staging`, and not in `prod`? I think if we want to turn the flags on in prod, we should use the values in the saved file?

Reply:
@obulat we should make the flag set to disabled instead of switchable in the prod environment so that it cannot be turned on by the feature flag query params.